src/components/navMenu/index.ts

```diff
@@ -0,0 +1 @@
+export { NavMenu } from './NavMenu'
```

Comment:
Do we also need to export this in the top-level `index.tsx`?

Reply:
Oh right, thanks!

Reply:
Done: https://github.com/j2-health/design-system/pull/51/files#diff-b211563a379985cf25e1ffc4e40fd9bcbb669c8d4c61f4088b32fcd351c2722b